src/main/java/network/radicle/jetbrains/radiclejetbrainsplugin/services/RadicleProjectApi.java

```diff
@@ -83,7 +84,7 @@ public SeedNodeInfo checkApi(SeedNode node) {
 
     public List<RadIssue> fetchIssues(String projectId, GitRepository repo) {
         var node = getSeedNode();
-        var url = node.url + "/api/v1/projects/" + projectId + "/issues";
+        var url = node.url + "/api/v1/projects/" + projectId + "/issues?page=0&perPage=" + ALL_IN_ONE_PAGE;
```

Comment:
This API request will fetch only the issues with the state OPEN. So in order to get all the issues you have to make one more request with the state closed. So the URL will be 
` var url = node.url + "/api/v1/projects/" + projectId + "/issues?page=0&state=closed&perPage=" + ALL_IN_ONE_PAGE;`

You can check the available states from here 
https://github.com/cytechmobile/radicle-jetbrains-plugin/blob/2c989d04977645dbec2b525a6343201900aac315/src/main/java/network/radicle/jetbrains/radiclejetbrainsplugin/models/RadIssue.java#L55